src/main/resources/db/migration/all/20240502000000__create_cas_2_summary_view.sql

```diff
@@ -0,0 +1,20 @@
+-- create application summary view
+CREATE OR REPLACE VIEW cas_2_application_summary AS SELECT
+    CAST(a.id AS TEXT),
+    a.crn,
+    a.noms_number,
+    CAST(a.created_by_user_id AS TEXT),
+    nu.name,
+    a.created_at,
+    a.submitted_at,
+    a.hdc_eligibility_date,
+    asu.label,
+    CAST(asu.status_id AS TEXT),
+    a.referring_prison_code
+FROM cas_2_applications a
+LEFT JOIN (SELECT DISTINCT ON (application_id) su.application_id, su.label, su.status_id
+    FROM cas_2_status_updates su
+    ORDER BY su.application_id, su.created_at DESC) as asu
+    ON a.id = asu.application_id
+JOIN nomis_users nu ON nu.id = a.created_by_user_id
+WHERE (a.conditional_release_date IS NULL OR a.conditional_release_date >= current_date)
```

Comment:
Also regarding whether `conditional_release_date IS NULL` is required, it is because this query is also used to return all (in progress and submitted applications) for the assessor dashboard.  In progress applications where CRD will be null need to be included here.